projects/core/src/cart/store/effects/cart.effect.ts

```diff
@@ -95,7 +93,9 @@ export class CartEffects {
     private productImageConverter: ProductImageConverterService,
     private occCartService: OccCartService,
     private cartData: CartDataService
-  ) {}
+  ) {
+    this.cartData.getDetails = true;
```

Comment:
why are we setting this to true?

Reply:
That param set as false caused fetching incomplete data

Reply:
but what if we actually need `getDetails` to be false?

Reply:
@tobi-or-not-tobi can you take a look?